index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Promise: Portfolio</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <!-- first-section -->
+  <div class="wrapper">
+    <div class="mobile-view">
+      <div class="header-wrapper">
+        <!-- navbar-section -->
+        <header class="header-section">
+          <span class="nav-span">Promise Okechukwu</span>
+          <span><a href="#"><img src="./images/Enabled.png" class="#" alt="#"></a></span>
+         
+          <article class="nav-links">
+            <h3>
```

Comment:
- [ ] Note that these are not the headings. There is no need to wrap them into an `h3` tag. These link tags should be list items and wrapped into a `ul` tag. Please correct that.